### ISSUE TITLE
Funnels Inconsistant and Unrealistic

### ISSUE THREAD

**Issue by bdjnk** [CleverRaven/Cataclysm-DDA#38713]
# Describe the bug

 A regular (plastic) funnel is 606g and 500ml with a 300mm radius. A metal funnel is 4017g and 4L with a 400mm radius. Meanwhile hand-made funnels are 4g and 250ml with a 200mm radius.

Those radii are exaggerated. 400mm is 15.75in. The largest funnels I could find have a diameter of 10in or less, which is a radius of 5in or 127mm. However, those large funnels are less common than smaller 6in diameter funnels.

Those weights are inconsistent, with the factory funnels too heavy and the hand-made funnels much too light. The largest heaviest steel funnels I could find were in the 1.4lbs range or 635g, while the largest heaviest plastic funnels were in the 0.55lbs range or 250g. Meanwhile, hand-made items are often heavier than factory designs, and 4g is clearly too light for anything functioning as a funnel.

# Steps To Reproduce

Use funnels. Notice the weights are off. Investigate and discover the radii are also off.

# Expected behavior

Funnels in the game should reasonably approximate funnels in real life.

# Additional context

Since funnels are a major source of potable water, it might be tempting not to alter them such that they become less effective. I will open a separate feature request regarding carbon filters that should mitigate any such concerns.

**Comment by Funguss** [comment]
Well, I've not bothered with funnels in years, so take this with a pinch of salt, but... I'd have thought the makeshift ones were made from some kind of tarp to keep them lightweight. IIRC they're made from plastic chunks and the recipe dates back to original Cata, which (again IIRC) didn't have plastic sheets or tarpaulin, so it's just an overlooked recipe that comes from a time when these were abstracted. Also, with the upcoming chemistry overhaul I expect funnels to start having greater functionality, as they are they're pretty much a gimmick. Not tried them with the recent weather changes but I expect they've impacted them pretty heavily, too.

tl;dr Nerf 'em, but buff makeshift funnels or totally alter/remove the recipe now we have tarp rain catchers. Unless the chemistry overhaul will use them.

**Comment by Maleclypse** [comment]
Are funnels still a major source of water?  Since there are only roughly three rainy days per season now?
